stdlib/LinearAlgebra/test/generic.jl

```diff
@@ -248,6 +253,31 @@ end
     end
 end
 
+@testset "normalize for multidimensional arrays" begin
+    function test_arr(arr)
+        @test normalize(arr) == normalize!(copy(arr))
+        @test size(normalize(arr)) == size(arr)
+        @test vec(normalize(arr)) == normalize(vec(arr))
+    end
+
+    arr = [ 1.0 0.0; 0.0 1.0 ]
+    test_arr(arr)
+
+    arr = [
+            [1.0 0.0 0.0];
+            [0.0 1.0 0.0]
+         ]
```

Comment:
`arr = [ 1.0 0.0 0.0; 0.0 1.0 0.0 ]` or
```
arr = [ 1.0 0.0 0.0
            0.0 1.0 0.0 ]
````

Reply:
removed test case in  cdb2167